src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -65,7 +66,7 @@ private static async Task TestWithOptionsAsync(TestWorkspace workspace, params A
 
         private static async Task TestWithOptionsAsync(Document document, QuickInfoService service, int position, Action<QuickInfoItem>[] expectedResults)
         {
-            var info = await service.GetQuickInfoAsync(document, position, cancellationToken: CancellationToken.None);
+            var info = await service.GetQuickInfoAsync(document, position, SymbolDescriptionOptions.Default, CancellationToken.None);
```

Comment:
aside: i know its' test code, but would it not make sense to still do `SymbolDescriptionOptions.From(document.Project);` here?

Reply:
These options will be stored in global options, not in solution snapshot. If the test needs to set those options they will then read it from global options.